docker-compose.kafka.yml

```diff
@@ -1,5 +1,5 @@
 mysql:
-  image: mysql:5.7
+  image: mysql:5.6
```

Comment:
Curious what's the reason to downgrade mysql?

Reply:
To match the mysql version of RDS, I was testing the migration locally with docker